templates/a0-widget-setup-form.php

```diff
@@ -4,115 +4,131 @@
 </p>
 <p>
-	<label for="<?php echo $this->get_field_id( 'dict' ); ?>"><?php _e( 'Translation:' ); ?></label>
+	<label for="<?php echo $this->get_field_id( 'dict' ); ?>"><?php _e( 'Translation:', 'wp-auth0' ); ?></label>
 	<textarea class="widefat" id="<?php echo $this->get_field_id( 'dict' ); ?>"
-			  name="<?php echo $this->get_field_name( 'dict' ); ?>">
-		<?php echo esc_attr( $dict ); ?>
-	</textarea>
-	<br/><span class="description">
-			<?php echo __( 'This is the widget\'s dict param.', 'wp-auth0' ); ?>
-		<a target="_blank" href="https://github.com/auth0/lock/wiki/Auth0Lock-customization#dict-stringobject"><?php echo __( 'More info', 'wp-auth0' ); ?></a>
-		</span><br>
-		<span class="description">
-			<i><b><?php echo __( 'Note', 'wp-auth0' ); ?>:</b>
-				<?php echo __( 'This can override the "Form title" setting', 'wp-auth0' ); ?>
-			</i>
+			  name="<?php echo $this->get_field_name( 'dict' ); ?>"><?php echo $dict; ?></textarea>
+	<br><span class="description">
+			<?php _e( 'The languageDictionary parameter for the Auth0 login form. ', 'wp-auth0' ); ?>
+	</span>
+	<br><span class="description">
+			<?php
+			printf(
+				'<a href="https://github.com/auth0/lock/blob/master/src/i18n/en.js" target="_blank">%s</a>',
+				__( 'List of all modifiable options', 'wp-auth0' )
+			)
+			?>
 		</span>
+	<br><span class="description">
+			<?php
+			_e( 'NOTE: This field is deprecated and will be removed in the next major release. ', 'wp-auth0' );
+			_e( 'Use a languageDictionary property the Extra Settings field below to change text.', 'wp-auth0' );
+			?>
 	</span>
 </p>
 <p>
-	<label for="<?php echo $this->get_field_id( 'extra_conf' ); ?>"><?php _e( 'Extra configuration:' ); ?></label>
+	<label for="<?php echo $this->get_field_id( 'extra_conf' ); ?>"><?php _e( 'Extra Settings', 'wp-auth0' ); ?></label>
 	<textarea class="widefat" id="<?php echo $this->get_field_id( 'extra_conf' ); ?>"
-			  name="<?php echo $this->get_field_name( 'extra_conf' ); ?>">
-		<?php echo esc_attr( $extra_conf ); ?>
-	</textarea>
-	<br/><span class="description">
-			<?php echo __( 'This field allows you to set all the widget settings.', 'wp-auth0' ); ?>
-		<a target="_blank" href="https://github.com/auth0/lock/wiki/Auth0Lock-customization"><?php echo __( 'More info', 'wp-auth0' ); ?></a>
-		</span><br>
-		<span class="description">
-			<i><b><?php echo __( 'Note', 'wp-auth0' ); ?>:</b>
-				<?php echo __( 'The other settings will override this configuration', 'wp-auth0' ); ?>
-			</i>
+			  name="<?php echo $this->get_field_name( 'extra_conf' ); ?>"><?php echo $extra_conf; ?></textarea>
+	<br><span class="description">
+			<?php _e( 'Valid JSON for Lock options configuration; will override all options set elsewhere.', 'wp-auth0' ); ?>
+		<a target="_blank" href="https://auth0.com/docs/libraries/lock/v11/configuration"><?php _e( 'See options and examples', 'wp-auth0' ); ?></a>
 		</span>
-	</span>
 </p>
+
+<?php if ( $custom_css ) : ?>
```

Comment:
Do not show the field if empty to avoid new data to be saved (this option is deprecated).